README.md

```diff
@@ -1 +1,7 @@
 [![Review Assignment Due Date](https://classroom.github.com/assets/deadline-readme-button-22041afd0340ce965d47ae6ef1cefeee28c7c493a6346c4f15d667ab976d596c.svg)](https://classroom.github.com/a/GeL61fu8)
+
+## Credentials to log in the db:
+
+1. name: Stef
+2. stef@hotmail.com
+3. Stef123+
```

Comment:
I assume you're not using this password anywhere else but just as a rule of thumb: never add credentials to source code 🙂